doc/usage.rst

```diff
@@ -141,11 +157,13 @@ few command line options:
 
    Use setuptools instead of distribute in the virtualenv
 
-.. cmdoption:: --no-test
+.. cmdoption:: --setuptools-test
+
+   .. versionadded:: 1.0
 
-   Skip running ``python setup.py test`` after dependencies and the
-   package is installed. This is useful if the Python code is packaged
-   using distutils and not setuptools.
+   Force running ``python setup.py test`` when building the package.
```

Comment:
I wouldn't say "force running" here but simply "run". Forcing suggests there's a need to apply some pressure to get this done whereas we simply flipped the default.